src/main/java/reposense/report/SummaryJson.java

```diff
@@ -41,7 +42,7 @@ public SummaryJson(List<RepoConfiguration> repos, ReportConfiguration reportConf
         this.isUntilDateProvided = isUntilDateProvided;
         this.repoSenseVersion = repoSenseVersion;
         this.errorSet = errorSet;
-        this.zoneId = zoneId;
+        this.zoneId = zoneId.toString();
```

Comment:
Related to the above comment, this `toString()` conversion looks out of place relative to the other variable initializations.

Reply:
Actually I took a look, there are also other variables similar to this

``` java
this.reportTitle = reportConfig.getTitle();
// and
this.supportedDomainUrlMap = SupportedDomainUrlMap.getDefaultDomainUrlMap();
```

Reply:
Switched back to `this.zoneId = zoneId;`